src/fileInfo.hpp

```diff
@@ -0,0 +1,19 @@
+#include <vector>
+#include <string>
+#include <fstream>
+
+using namespace std;
+
+vector<char> readF(string file)
```

Comment:
```suggestion
vector<char> readF(const string &file)
```
Объекты, которые не будут меняться лучше передавать по констанстной ссылке. Так мы избежим глубокого копирования. (см. способы передачи аргументов)
Если есть вопросы по ссылкам - разберем.